3-reveal/migrations/5-IRS/4-Zambia-2020/sqitch.plan

```diff
@@ -14,3 +14,5 @@ zambia_irs_sop_date_performance [zambia_irs_spray_event zambia_daily_summary_eve
 zambia_irs_mop_up 2020-10-02T10:00:50Z gstuder,,, <gstuder@gstuder-ThinkPad-T490> # Changes for Zambia IRS mop-up reports
 zambia_irs_structures_geojson [reveal_irs_zambia_2019:zambia_irs_structures] 2020-10-15T16:08:43Z mosh <kjayanoris@ona.io> # Add zambia_irs_structures_geojson.
 remove_hard_coded_jurisdiction_depth [reveal_database_views:jurisdictions_tree] 2020-11-05T15:22:32Z mosh <kjayanoris@ona.io> # Remove hard-coded jurisdiction depth.
+zambia_focus_area_irs_base [reveal_transaction_tables:plans reveal_irs_zambia_2019:zambia_plan_jurisdictions reveal_irs_zambia_2019:zambia_jurisdictions reveal_irs_zambia_2019:zambia_irs_structures utils:count_elements utils:array_concat_agg] 2020-11-30T14:28:46Z Ona,,, <ona@ona-ThinkPad-T470> # Redo zambia irs mop up
+redo_zambia_irs_mop_up [zambia_focus_area_irs_base] 2020-11-30T14:30:32Z Ona,,, <ona@ona-ThinkPad-T470> # Redo zambia irs mop up
```

Comment:
I think we should use another name.  it is not really accurate to say `redo_zambia_irs_mop_up`, is it?